tools/indextemplate/indextemplate_test.go

```diff
@@ -295,17 +295,17 @@ func TestHandleIndexBad(t *testing.T) {
 		{
 			label:    "manifests is not a JSON array",
 			response: `{"manifests": {}}`,
-			expected: "json: cannot unmarshal object into Go value of type []v1.Descriptor",
+			expected: "json: cannot unmarshal object into Go struct field Index.manifests of type []v1.Descriptor",
```

Comment:
I think these are due to changing versions (I developed the tests with Go 1.7.4).  We probably want to use regexps instead of fixed strings.  I'll put that into a rerolled #29.

Reply:
>  We probably want to use regexps instead of fixed strings.

[Done][1].

[1]: https://github.com/xiekeyang/oci-discovery/pull/29#issuecomment-331215096